src/router/routes.js

```diff
@@ -262,6 +262,14 @@ module.exports = [
       auth: false
     }
   },
+  {
+    name: 'oms.bodies.boards',
+    path: '/bodies/:id/boards',
+    component: 'core/bodies/ViewBoards',
```

Comment:
Why is this not part of network?

Reply:
Apparently it's easier this way due to the ChangeBoardModal component and the structure in the folders.